src/main/java/com/spotify/reaper/core/HostMetrics.java

```diff
@@ -0,0 +1,80 @@
+package com.spotify.reaper.core;
+
+import javax.annotation.Generated;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove this unused import 'javax.annotation.Generated'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)